app/views/games/index.html.erb

```diff
@@ -1,14 +1,5 @@
-<p style="color: green"><%= notice %></p>
+<h1 style="text-align: center">Games</h1>
```

Comment:
I am thinking to align this left rather than center. And then maybe add the `New Game` button on the same line, aligned right 🤔 

Can we try to see how it'd look?

Reply:
<img width="1651" alt="image" src="https://github.com/robert-damoc/game-stats/assets/137196597/f7e0728e-ace2-4e89-9227-209e092acf68">

Reply:
I think this looks a bit better, wdyt?